be/src/service/service_be/lake_service.cpp

```diff
@@ -471,8 +471,8 @@ void LakeServiceImpl::delete_txn_log(::google::protobuf::RpcController* controll
         return;
     }
 
-    if (request->txn_ids_size() == 0) {
-        cntl->SetFailed("missing txn_ids");
+    if (request->txn_ids_size() == 0 && request->txn_infos_size() == 0) {
+        cntl->SetFailed("missing txn_ids and txn_infos");
```

Comment:
It is the `txn_ids` and `txn_infos` can't be both empty. the error message reads like the txn_ids and txn_infos both must be set.